content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -179,7 +189,7 @@ To define the `UserInfo` view from the example above, begin by using the `view`
 
 ```prisma
 view UserInfo {
-  // Fields
+// Fields
```

Comment:
Your formatting tool does not understand `view` yet I fear. What are you using?

Reply:
This happens when the husky pre-commit hook runs `npx lint-staged`. To get around it I usually have to comment out that line, commit, and then uncomment it.